library/src/main/java/co/moonmonkeylabs/realmrecyclerview/RealmRecyclerView.java

```diff
@@ -53,6 +57,12 @@
     private int gridWidthPx;
     private boolean swipeToDelete;
     private int bufferItems = 3;
+    private Orientation orientation;
+    private int getOrientationValue() {
```

Comment:
Perhaps this could be named `getLayoutManagerOrientation`. It's tied to `LinearLayoutManager` at them moment. 
Also could you move it below the `init` method.